plugins/.gitignore

```diff
@@ -0,0 +1,4 @@
+[^.]*
+!__init__.py
+!upload.py
+!hello.py
```

Comment:
need to restore ```VERSION``` file or else 'pip install' won't work anymore for slackbot when it's packaged up (or installed from docker)

I think we should scrap the idea of moving the plugins dir to the root dir in the core repo and just leave it where it is